### ISSUE TITLE
CloudFlare passes

### ISSUE THREAD

**Issue by MikeyBoo01** [privacypass/challenge-bypass-extension#249]
Before filing a new bug, verify if your request is already being tracked by another issue here: https://github.com/privacypass/challenge-bypass-extension/issues.

--- 

If you believe that this is a new bug, please proceed to create an issue. The issue will be investigated after you have filled in the following information.

**Describe the bug**

CloudFlare passes are not adding.

**To Reproduce**
1. Go to the Privacy Pass extension.
2. Click on CloudFlare to get more passes.
3. Follow the procedure, click on the captcha and solve it.
4. No passes are added (and I even got some hCaptcha even though it was not the case).

**Expected behavior**

CloudFlare passes should add to the extension, but none appears.

**System (please complete the following information):**
 - OS: Windows 10
 - Cloudflare tokens
 - Google Chrome
 - Chrome version: 93.0.4577.63
 - Privacy Pass Version: 2.0.9
 - Installed from Chrome Web Store

**Additional context**

Before: https://i.imgur.com/7FBkQjP.png
During the captcha: https://i.imgur.com/Z7raFa3.png
After: https://i.imgur.com/Y5Deqpg.png

My first time adding CloudFlare did work, but not anymore.

**Comment by ppopth** [comment]
I cannot reproduce this one yet. Does it happen all the time or only sometimes?

**Comment by MikeyBoo01** [comment]
> I cannot reproduce this one yet. Does it happen all the time or only sometimes?

Hey there @ppopth ! This issue is happening all the time to me, no Cloudflare passes are adding, no matter how hard I try. I've read something similar in: https://github.com/privacypass/challenge-bypass-extension/issues/212 .

**Comment by NightPlay** [comment]
I had this problem but it got fixed next time, I kept waiting on "Redirecting" and it gave me passes.

**Comment by Silur** [comment]
same issue for over a year, cloudflare functionality is basically nonexistent

**Comment by MikeyBoo01** [comment]
@Silur Very much agree... :(

**Comment by aGrimRepoMan** [comment]
As an added data point, I'm seeing the same issue @MikeyBoo01 describes, except on Firefox browser.  Reproducible on every attempt. Config as follows:
    OS: Windows 10
    Cloudflare tokens
    Mozilla Firefox, version 92.0
    Privacy Pass Version: 2.0.9
    Installed from https://addons.mozilla.org/

**Comment by CJYKK** [comment]
Same Issue!

**Comment by SpinB** [comment]
Same issue on Firefox v92.0.1.

**Comment by Mikowmer** [comment]
Having the same issue on my end:
OS: Windows 10
Cloudflare Tokens
Microsoft Edge (Chromium), version  94.0.992.31
Privacy Pass version 2.0.9
Installed from Chrome Web Store

**Comment by ValZapod** [comment]
Redirecting is not supposed to work. Anyway, it works here.
